extensions.tex

```diff
@@ -40,21 +41,50 @@ \subsection{Beta Distribution Bag-of-words}
 \end{table}
 
 
-  \subsection{BoW + NN}
+  \subsection{Bag-of-words Feature Kernel Neural Net}
   \label{sec:bow_nn}
-
+  The bag-of-words predictors described in Sections \ref{sec:bag_of_words} and \ref{sec:bow_beta} perform excellently in terms of accuracy, but the loss (observable only when utilising beta distributions) is very poor. Conversely the windowed neural net predictor in Section \ref{sec:win_nn} has a comparably poor accuracy but performs vastly better with regards to loss.
+  
+  This is because the neural network is directly trained against the loss function, whereas the bag-of-words models are not. Combining these two approaches -- the simplicity of a bag-of-words model with the loss-optimising nature of a neural net -- is likely to provide the best of both worlds with regards to loss and accuracy metrics.
+  
+  The beta-distribution bag-of-words predictor utilises solely the product of probabilities to form its prediction. While a classifier could be built on top of this alone, more metrics may be offered as input to improve performance. In this case the following are provided to the input layer, for each author:
+  
+  \begin{itemize}
+  \item The mean log probability of all words \footnote{For any given sentence, this is simply a length-scaled version of the log likelihood. However by taking the mean, the magnitude of these variables is not dependent on sentence length}
+  \item The max and min log probability of all words
+  \item The standard deviation of log probabilities of all words
+  \item The number of "missed" words in the sentence \footnote{"Missed" words are those not observed as having been used by an author in the training set. Whilst they are still assigned a probability as in Section \ref{sec:bow_beta}, their record as having been missed is retained}
+  \end{itemize}
+  
+ Additionally, the sentence length is included based on the observations in Section \ref{sec:sentence_lengths}. This provides a 16 neuron input-layer.
+ 
+ The confusion matrix, accuracy and loss for this model may be seen in Table \ref{tab:bow_nn_res}. The results presented utilise a perceptron with no hidden layers; adding hidden layers does not improve performance. Noticeable overfitting is observed with this architecture. Its training loss is $<0.2$, and its training accuracy $>90%$. Attempts to overcome this such as compressing the model through a 2-neuron layer and increasing batch sizes were not successful.
```

Comment:
compile error: missing an escape on the percent sign `$>90%$` should be  `$>90\%$`